assets/src/edit-story/app/story/effects/useLoadStory.js

```diff
@@ -89,6 +90,14 @@ function useLoadStory({ storyId, shouldLoad, restore }) {
         const pages =
           storyData?.pages?.length > 0 ? storyData.pages : [createPage()];
 
+        // Initialize color presets, if missing.
+        if (!stylePresets.colors) {
```

Comment:
Are these generic "color" presets? Or specifically "fill" presets? If "fill", maybe it's best to call them `fillColors` to follow your explicit `textColors`?

Reply:
These names are mainly taken based on the division in Figma, `fillColor` might be more accurate here indeed!